### PR TITLE
Missing changes of PR #3927

### PR DESCRIPTION
**Pull request by marcosgdf**
Missing changes of PR #3927

### CODE REVIEW

**Comment by eldy** [comment]
?? You remove an important commit ?
If you add constant PRODUIT_MULTIPRICES_ALLOW_AUTOCALC_PRICELEVEL, you should see the new feature without your PR, don't you ?

**Comment by marcosgdf** [comment]
¿? I think you are confusing this PR with another one. It is a commit that got lost in PR #3767

**Comment by eldy** [comment]
I dont think so.
When i look at changed files in your PR, for exemple, i see the line
dol_banner_tab($object, 'ref', '', ($user->societe_id?0:1), 'ref');
that is removed in file htdocs/product/price.php.

And a lot of other code that must remain are removed. Also i don't see what the PR bring.

**Comment by marcosgdf** [comment]
Corrected. Those changes weren't intended.

File: htdocs/product/price.php [hunk 1]
```diff
@@ -874,16 +875,21 @@ function on_change() {
...
-			print '<td><input name="price_min" size="10" value="' . price($object->price_min) . '">';
+			print '<input name="price_min" size="10" value="' . price($object->price_min) . '">';
 		}
 		if ( !empty($conf->global->PRODUCT_MINIMUM_RECOMMENDED_PRICE))
 		{
-			print '<td align="left">'.$langs->trans("MinimumRecommendedPrice", price($maxpricesupplier,0,'',1,-1,-1,'auto')).' '.img_warning().'</td>';
+			print '</td><td align="left">'.$langs->trans("MinimumRecommendedPrice", price($maxpricesupplier,0,'',1,-1,-1,'auto')).' '.img_warning().'</td>';
 		}
-		print '</td></tr>';
+		print '</tr>';
```
Comment (eldy) [inline htdocs/product/price.php:893]: The closing td is included in the previous if, but if the if is false, no closing td is done

Comment (marcosgdf) [inline htdocs/product/price.php:893]: Done!

**Pull request merged**